pages/admin/Dashboard.js

```diff
@@ -0,0 +1,69 @@
+import styled from "@emotion/styled";
+import React, { useEffect } from "react";
+import { useRouter } from "next/router";
+import { Button, Grid } from "@mui/material";
+const ContentContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 100px;
+`;
+const ContentWrapper = styled.div`
+  display: flex;
+  padding: 20px;
+  gap: 40px;
+`;
+const Tabs = styled(Button)({
+  display: "flex",
+  justifyContent: "spaceEvenly",
+  alignItems: "center",
+  width: "250px",
+  height: "100px",
+  backgroundColor: "#dad4d4",
+  borderRadius: "10px",
+  cursor: "pointer",
+  boxShadow: "3px 0px 6px 0px rgba(246, 246, 246, 0.75)",
+});
+
+const Categories = styled.div`
+  color: rgb(70, 105, 70);
+`;
+
+function Dashboard() {
+  const router = useRouter();
+  const handleClickUser = (e) => {
+    e.preventDefault();
+    router.push("http://localhost:3000/admin/UsersActive");
+  };
+  const handleClickPost = (e) => {
+    e.preventDefault();
+    router.push("http://localhost:3000/admin/post");
+  };
```

Comment:
The redirect URL here should be changed to page's origin + page route (e.g. window.location.origin + '/admin/...'). So when we deploy code to a host, we don't need to change code again.